docs/SG-NEx_xPore_tutorial.md

```diff
@@ -0,0 +1,191 @@
+
+- feature values from both the A549 and HepG2 samples. (two json files, one from each sample)
+- indexes of the feature values file. (two index files, one from each sample)
+
+### **Download data for xPore**
+The data files can be downloaded from the [SG-NEx AWS S3 bucket](http://sg-nex-data.s3-website-ap-southeast-1.amazonaws.com/). Please refer to this [page](https://github.com/GoekeLab/sg-nex-data/blob/update-docs-aws/docs/AWS_data_access_tutorial.md) for a comprehensive tutorial to access the SG-NEx dataset.
+
+To begin, we will create directories to store the required files for the respective samples. 
+```bash
+# create directories for A549 and HepG2 samples
+mkdir A549_directRNA_replicate6_run1
+mkdir HepG2_directRNA_replicate6_run1
+```
+
+Then, we will access the S3 bucket to download the data into the created directories. There are two methods to download the data. 
+<br>
+
+#### **Method One(Recommended):** 
+Note: The command requires you to have [AWS CLI](https://aws.amazon.com/cli/) installed.
+
+**For A549 sample:**
+```bash
+# To download the processed data for A549 sample
+aws s3 sync --no-sign-request s3://sg-nex-data/data/processed_data/xpore/SGNex_A549_directRNA_replicate6_run1 ./A549_directRNA_replicate6_run1
+# The directory should now contain two files. They are the json and index files.
+```
+
+**For HepG2 sample:**
+```bash
+# To download the processed data for HepG2 sample
+aws s3 sync --no-sign-request s3://sg-nex-data/data/processed_data/xpore/SGNex_HepG2_directRNA_replicate6_run1 ./HepG2_directRNA_replicate6_run1
+# The directory should now contain two files. They are the json and index files.
+```
+
+Alternatively, we can use the URL to the S3 bucket to download the processed data. 
+<br>
+
+#### **Method Two:**
+**For A549 sample:**
+```bash
+cd A549_directRNA_replicate6_run1 
+
+# To download the json file
+wget https://sg-nex-data.s3.ap-southeast-1.amazonaws.com/data/processed_data/xpore/SGNex_A549_directRNA_replicate6_run1/data.json
+
+# To download the index file
+wget https://sg-nex-data.s3.ap-southeast-1.amazonaws.com/data/processed_data/xpore/SGNex_A549_directRNA_replicate6_run1/data.index
+
+# Exit the A549 directory to return to our working directory
+cd ..
+```
+
+**For HepG2 sample:**
+```bash
+cd HepG2_directRNA_replicate6_run1 
+
+# To download the json file
+wget https://sg-nex-data.s3.ap-southeast-1.amazonaws.com/data/processed_data/xpore/SGNex_HepG2_directRNA_replicate6_run1/data.json	
+
+# To download the index file
+wget https://sg-nex-data.s3.ap-southeast-1.amazonaws.com/data/processed_data/xpore/SGNex_HepG2_directRNA_replicate6_run1/data.index
+
+# Exit the HepG2 directory to return to our working directory 
+cd ..
+```
+<br>
+
+Then, we need to prepare a `YAML` file to specify the experimental design, the data directories, the output directory, and the method options. We can create a `.yml` file using the command below: 
```

Comment:
```suggestion
Then, we need to prepare a `YAML` file to specify the model configuration including experimental design, the data directories, the output directory, and the method options. We can create a `.yml` file using the command below: 
```

Reply:
I think experimental design cover all the specification ?

Reply:
I think it makes sense, can ignore this one also then.